### PR TITLE
x11-themes/sugar-candy: New package

### PR DESCRIPTION
**Pull request by Kangie**
Sugar-Candy is a pretty, lightweight, clean, and simple theme for SDDM.

Closes: https://bugs.gentoo.org/693462
Signed-off-by: Matt Jolly <Kangie@footclan.ninja>
Package-Manager: Portage-2.3.89, Repoman-2.3.20

### CODE REVIEW

File: x11-themes/sugar-candy/sugar-candy-1.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="svg jpeg"
+
+# Require at least one kind of image support. Kind of useless otherwise.
+REQUIRED_USE="!jpeg? ( svg )
+!svg? ( jpeg )"
+
+DEPEND=">=dev-qt/qtquickcontrols2-5.11
+svg? ( >=dev-qt/qtsvg-5.11 )
```
Comment (thesamesam) [inline x11-themes/sugar-candy/sugar-candy-1.6.ebuild:21]: Indentation

File: x11-themes/sugar-candy/sugar-candy-1.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+src_install() {
+	local DOCS=( AUTHORS COPYING CHANGELOG.md README.md )
+	einstalldocs
+	local target="${ROOT}/usr/share/sddm/themes/${PN}"
+	insinto ${target}
+	doins -r *
+}
+
+pkg_postinst () {
```
Comment (thesamesam) [inline x11-themes/sugar-candy/sugar-candy-1.6.ebuild:39]: Maybe just show this on first install by checking ${REPLACING_VERSIONS}.

File: x11-themes/sugar-candy/sugar-candy-1.6.ebuild [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+>=dev-qt/qtgraphicaleffects-5.11
+>=dev-qt/qtdeclarative-5.11
+>=dev-qt/qtgui-5.11[jpeg?]
+>=x11-misc/sddm-0.18"
+RDEPEND="${DEPEND}"
+BDEPEND=""
+
+S="${WORKDIR}/sddm-sugar-candy-${PV}"
+
+src_install() {
```
Comment (thesamesam) [inline x11-themes/sugar-candy/sugar-candy-1.6.ebuild:31]: I guess the default Makefile or whatever does not install for you?

File: x11-themes/sugar-candy/sugar-candy-1.6.ebuild [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+	local DOCS=( AUTHORS COPYING CHANGELOG.md README.md )
```
Comment (thesamesam) [inline x11-themes/sugar-candy/sugar-candy-1.6.ebuild:32]: Usually, this is put in global scope.

File: x11-themes/sugar-candy/sugar-candy-1.6.ebuild (hunk 3 above)
Comment (Kangie) [reply x11-themes/sugar-candy/sugar-candy-1.6.ebuild:31]: Upstream installation instructions are (paraphrasing) "download a tar of the current version of the repo and extract to the sddm themes directory". I could create a makefile, but the install is trivial.

**Comment by ajakk** [comment]
Seems like no one is there anymore.

In any case please add new packages to [GURU](https://wiki.gentoo.org/wiki/Project:GURU) rather than straight to ::gentoo.

**Pull request closed without merging**